src/shared/credentials/defaultCredentialSelectionDataProvider.ts

```diff
@@ -186,19 +186,19 @@ export class DefaultCredentialSelectionDataProvider implements CredentialSelecti
     }
 }
 
-export async function credentialProfileSelector(
+export async function credentialSourceSelector(
     dataProvider: CredentialSelectionDataProvider
 ): Promise<CredentialSelectionState | undefined> {
-    async function pickCredentialProfile(
+    async function pickCredentialSource(
```

Comment:
Even if we decide to expose this concept as a "credentials source" in the UI (although we [decided against that](https://github.com/aws/aws-toolkit-vscode/pull/1725#discussion_r655686537) recently),  not sure we should rename these internal symbols. "profile" is the common case, and technically the value is a `source:profile` pair.

Reply:
> If we are confident in the `source:profile` pairing we should consider baking that concept through to `CredentialsId` - `credentialTypeId` should be renamed to `credentialProfile`.

I'm not confident but I agree we need to commit :) Let's do it....